ajax/libs/reactstrap/package.json

```diff
@@ -24,7 +24,7 @@
     {
       "basePath": "dist",
       "files": [
-        "reactstrap.min.js*"
+        "reactstrap(.full)?(.min)?.js*"
```

Comment:
This rule can not fetch any file under `dist`. 
How about `**/reactstrap!(*es*|*cjs*)`?
`**/` is for recursively fetching the files under `dist`.
So `**/reactstrap!(*es*|*cjs*)` would be recursively fetching all `reactstrap` related files except for the ones involve `es`or `cjs`.

Reply:
I made the change you suggested, though it would seem like the current value `reactstrap.min.js*`  not having `**/` would have the same issue.

Reply:
@TheSharpieOne For the current file structure. They are indeed the same. Assuming the file structure changes, there is a sub-folder. `reactstrap.min.js*` will not let our bot fetch the files under sub-folder even they are required. But `**/reactstrap!(*es*|*cjs*)` can do this. Hope I didn't misunderstand what you are talking about and you understand what I am saying :)

Reply:
Gotcha. It is to better future-proof the glob in case sub-directories are added in the future.

Reply:
Exactly! :smile: